analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RoslynSymbolicExecution.cs

```diff
@@ -166,6 +166,10 @@ private ProgramState ProcessBranchState(ControlFlowBranch branch, ProgramState s
             {
                 state = state.RemoveCapture(capture);
             }
+            if (branch.Source.EnclosingRegion is { Kind: ControlFlowRegionKind.Catch or ControlFlowRegionKind.FilterAndHandler })
```

Comment:
You can add a comment mentioning that exception filtering will be added later.

Reply:
That part is not relevant for this line/method. The filtering will happen around `ExceptionSuccessors` method.